build.gradle.kts

```diff
@@ -54,6 +57,20 @@ tasks.publishPlugin {
     channels.set(listOf(project.version.toString().split('-').getOrElse(1) { "default" }.split('.').first()))
 }
 
+tasks {
+    withType<ShadowJar> {
+        archiveBaseName = "detekt-intellij-plugin-shaded"
+        mergeServiceFiles()
+        relocate("org.jetbrains.kotlin", "detekt.shadow.org.jetbrains.kotlin")
+    }
+}
+
+tasks.withType<PrepareSandboxTask> {
```

Comment:
I think this line triggers the shadowJar task eagerly.

`tasks.withType<PrepareSandboxTask>.configure {`